src/core/src/package_managers/AptitudePackageManager.py

```diff
@@ -182,9 +182,6 @@ def get_security_updates(self):
         code, out = self.env_layer.run_command_output(self.prep_security_sources_list_cmd, False, False)
         if code != 0:
```

Comment:
can we use a constant for the 0 value something like Constants.Status_Success, i think we have a constant in constant.py

Reply:
Checking code with 0 or some numeric value is used in many places, so this will include significant code change. Will have this in a separate PR